### PR TITLE
Layout Component added

### PR DESCRIPTION
**Pull request by aliturab99**

### CODE REVIEW

File: dashboard/src/App.css [hunk 1]
```diff
@@ -1,5 +1,7 @@
 .App {
   text-align: center;
+  background-color: #124b66 !important;
```
Comment (ServilancerOfficial) [inline dashboard/src/App.css:3]: Add this to Variable Please

File: dashboard/src/App.css [hunk 2]
```diff
@@ -36,3 +38,8 @@
     transform: rotate(360deg);
   }
 }
+
```
Comment (ServilancerOfficial) [inline dashboard/src/App.css]: Remove Extra Line

File: dashboard/src/index.js [hunk 3]
```diff
@@ -2,6 +2,8 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
+import "primereact/resources/themes/lara-light-cyan/theme.css";
+
```
Comment (ServilancerOfficial) [inline dashboard/src/index.js]: Remove Extra Line

**Review by ServilancerOfficial — CHANGES_REQUESTED** [review]
Please make sure not to make these changes

File: dashboard/src/App.css (hunk 2 above)
Comment (aliturab99) [reply dashboard/src/App.css]: Resolved

Comment (aliturab99) [reply dashboard/src/App.css]: Resolved

File: dashboard/src/index.js (hunk 3 above)
Comment (aliturab99) [reply dashboard/src/index.js]: Removed

File: dashboard/src/App.css (hunk 1 above)
Comment (aliturab99) [reply dashboard/src/App.css:3]: added

**Pull request merged**